app/controllers/device_story_comments_controller.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+class DeviceStoryCommentsController < ApplicationController
+  before_action :set_device_story
+  before_action :set_device_story_comment, only: %i(show edit update destroy)
+
+  def index
+    @device_story_comments = @device_story.device_story_comments
+  end
+
+  def new
+    @device_story_comment = @device_story.device_story_comments.build
+  end
+
+  def create
+    @device_story_comment = @device_story.device_story_comments.build(device_story_comment_params)
+    respond_to do |format|
+      if @device_story_comment.save && !@device_story_comment.spam?
+        format.html { redirect_to device_story_path(@device_story), notice: 'Comment successfully submitted!' }
+      else
+        flash[:error] = 'Could not submit your comment. Please, try again!'
```

Comment:
Would trying definitely help? Might be good to capture the error that surfaced and show something to the user based on that. Of course if it's spam being obscure makes some sense.